test/integration/src/test/java/io/pravega/test/integration/StreamCutsTest.java

```diff
@@ -125,18 +130,21 @@ public void testReaderGroupCuts() throws Exception {
         ReaderGroupManager groupManager = new ReaderGroupManagerImpl("test", controller, clientFactory,
                 connectionFactory);
         groupManager.createReaderGroup("cuts", ReaderGroupConfig
-                .builder().disableAutomaticCheckpoints().stream("test/test").build());
+                .builder().disableAutomaticCheckpoints().stream("test/test").groupRefreshTimeMillis(0).build());
```

Comment:
Specifying `groupRefreshTimeMillis(0)` when using `disableAutomaticCheckpoints()` is done to speed up the test, right?

Reply:
yes. It also makes it deterministic. Instead of relying on a clock and sleeping, it will check for updates to the state every time.